src/logic/operate.js

```diff
@@ -0,0 +1,19 @@
+import Big from './big.mjs';
+
+const operate = (n1, n2, op) => {
+  n1 = Big(n1);
+  n2 = Big(n2);
+  if (op === '+') {
+    return n1.plus(n2).valueOf();
+  } else if (op === '-') {
+    return n1.minus(n2).valueOf();
+  } else if (op === 'x') {
+    return n1.times(n2).valueOf();
+  } else if (op === '÷') {
+    return n1.div(n2).valueOf;
+  } else if (op === '%') {
+    return n1.div(100).valueOf();
+  }
```

Comment:
Hi, you should be using `switch` here, even though what you have will work but you have `if` `else` tree, switch is better.
- also watch out for the next milestone  dividing by `0` will blow up so you want to have a check to return error instead.

Reply:
Thank you very much I will make changes and then move on to last milestone.